STYLEGUIDE.md

```diff
@@ -224,6 +224,32 @@ def multiplex(text, count)
 end
 ```
 
+## Dynamic Dispatch
+
+Avoid calling `send` and its cousins unless you really need it. Metaprogramming can be extremely powerful, but in most cases you can write code that captures your meaning by being explicit:
+
+``` ruby
+# Bad 
```

Comment:
I know that this matches the language in the rest of the doc. But I'm a bit apprehensive about calling code that is *working* `Bad`. How about `Current` instead of `bad` and `Aspirational` instead of `good`.
A minor change to the comment, but I love the change.

Reply:
that's a very good point.

What about `Don't` and `Do` instead of `Bad` and `Good` to change it from being about the quality of something to being instructional? Maybe `Avoid` instead of don't, but I can't think of a good antonym for avoid here...

Reply:
How about _Avoid_ and _Prefer_?

Reply:
I'm going to go ahead and update it everywhere while I'm at it :)